.circleci/config.yml

```diff
@@ -64,6 +80,14 @@ references:
         export ALGOLIA_LIBRARY_INDEX_NAME="prod_docs_sourcer-library-reference"
         ./scripts/push-to-s3-prod.sh
 
+  create_releases_pr: &create_releases_pr
+    run:
+      name: create releases pr
+      command: ./scripts/create_releases_pr.sh
+      environment:
+        CONFIGURE_GIT: 0 # Tell the script that we're in CI and need to configure git for gruntwork-ci user
```

Comment:
Why `0`? Can we not pass a boolean?

Reply:
we can pass a string that says "true" or "false" but we'll have to treat it as a string. I'm fine either way.